util/signal_handler.cc

```diff
@@ -178,6 +188,20 @@ static void cause_crash()
 }
 #endif
 
+void emit_crash_metric(std::map<std::string, std::string> parameters, std::string_view host, std::string_view port)
+{
+  if (host.empty()) {
+    return;
```

Comment:
cerr: we didn't emit a metric because no host specified

Reply:
What about port?

Reply:
Port is defaulting to 4317, so I think that's ok.